### PR TITLE
Change

### PR DESCRIPTION
**Pull request by jmt177**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1792)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/it-was-me/it-was-me.md [hunk 1]
```diff
@@ -11,3 +11,5 @@ What was I to do next?
 [Dominate the world?](../hulk/smash.md)
 
 [Use super git powers to save the world?](../super-powers/practice-flying.md)
+
+[Chill out until the road is clear](../lighter/toke.md)
```
Comment (casey-collab) [inline english/it-was-me/it-was-me.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**